### ISSUE TITLE
The sheet is appearing when opening the keyboard

### ISSUE THREAD

**Issue by MickaelHrndz** [bnxm/sliding_sheet#56]
Hello,

When I'm opening the keyboard by tapping on a TextField, the sheet's header appears. It's as if `snapToExtent(SnapSpec.headerSnap)` was called.

**Comment by asideHazle** [comment]
I have the same issue

![sheet](https://user-images.githubusercontent.com/35303552/95983031-edd67600-0e5b-11eb-9a21-9e2f78738e40.gif)

```yaml
sliding_sheet: ^0.4.0
```

```bash
$ flutter doctor -v
[√] Flutter (Channel beta, 1.22.1, on Microsoft Windows [Version 10.0.19041.508], locale ko-KR)
    • Flutter version 1.22.1 at C:\src\flutter
    • Framework revision f30b7f4db9 (6 days ago), 2020-10-08 10:06:30 -0700
    • Engine revision 75bef9f6c8
    • Dart version 2.10.1

 
[√] Android toolchain - develop for Android devices (Android SDK version 30.0.2) 
    • Android SDK at C:\Users\Hskim\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java       
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe      

[√] Android Studio (version 4.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 48.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] VS Code (version 1.49.3)
    • VS Code at C:\Users\Hskim\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.15.0

[√] Connected device (4 available)
    • sdk gphone x86 arm (mobile) • emulator-5554 • android-x86    • Android 11 (API 30) (emulator)
    • Web Server (web)            • web-server    • web-javascript • Flutter Tools
    • Chrome (web)                • chrome        • web-javascript • Google Chrome 85.0.4183.121
    • Edge (web)                  • edge          • web-javascript • Microsoft Edge 86.0.622.38

• No issues found!
```

**Comment by bnxm** [comment]
Ok, thanks for the gif. I'll try to take a look at it in the coming days.

**Comment by asideHazle** [comment]
Reproducible code

```dart
import 'package:flutter/material.dart';
import 'package:sliding_sheet/sliding_sheet.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      home: MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  MyHomePage({Key key, this.title}) : super(key: key);
  final String title;

  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  SheetController controller;
  @override
  void initState() {
    controller = SheetController();
    super.initState();
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text(widget.title),
      ),
      body: SlidingSheet(
        controller: controller,
        duration: Duration(milliseconds: 200),
        closeOnBackButtonPressed: true,
        closeOnBackdropTap: true,
        snapSpec: const SnapSpec(
          initialSnap: 0,
          snap: true,
          snappings: const [
            SnapSpec.headerSnap,
            0.7,
          ],
          positioning: SnapPositioning.relativeToAvailableSpace,
        ),
        body: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: [
            TextField(),
            SizedBox(height: 20),
            RaisedButton(
              onPressed: () => controller.show(),
              child: Text('show'),
            ),
            RaisedButton(
              onPressed: () => controller.hide(),
              child: Text('hide'),
            ),
          ],
        ),
        headerBuilder: (context, state) {
          return Container(
            color: Colors.amber,
            child: Row(
              children: [
                Expanded(
                  child: RaisedButton(
                    onPressed: () => controller.expand(),
                    child: Text('expand'),
                  ),
                ),
                SizedBox(width: 20),
                Expanded(
                  child: RaisedButton(
                    onPressed: () => controller.collapse(),
                    child: Text('collapse'),
                  ),
                ),
              ],
            ),
          );
        },
        builder: (context, state) {
          return Container(
            color: Colors.grey[800],
            height: MediaQuery.of(context).size.height * 0.7,
            child: Center(
              child: Text('Sheet body'),
            ),
          );
        },
      ),
    );
  }
}

```

**Comment by bnxm** [comment]
Should be fixed with version 0.4.1
